pyproject.toml

```diff
@@ -30,7 +30,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 bitstring = ">=4.0.1"
-cdflib = "==1.2.3"
+cdflib = "==1.2.4"
```

Comment:
Can you revert this for now? because @bourque and I need to update other tests to be compliant with this new release. Once our tests are updated, then we can update to this new version.

Reply:
Come to think of it. May be keep cdflib versioin as you have and then add `@pytest.mark.xfail()` on test that fails that not related to Ultra. Then we can update tests afterwards. This way, anyone working on cdflib in the future will be up to date with latest version.